charts/auto-loans/origination-activity/figure-1a__number.html

```diff
@@ -0,0 +1,13 @@
+    
+<p class="block short-desc">
+    <strong>Source:</strong> <a href="https://github.com/cfpb/consumer-credit-trends/blob/master/data/num_data_AUT.csv">https://github.com/cfpb/consumer-credit-trends/blob/master/data/num_data_AUT.csv</a><br>
```

Comment:
Something to usability test in the future, to see which one is more helpful for our users? 😄